### DISCUSSION TITLE
The number of drivers per node affects upload speed

### DISCUSSION THREAD

**Discussion by wanganhong** [minio/minio#17668] [Polls]
## Context

I am testing the upload speed on a 3-node cluster, Noticeably slower upload speed when adding drive,what the matter?

"3 nodes with 10 disks per node"
The Put speed of 1 concurent: 10M/s

"3 nodes with 2 disks per node"
The Put speed of 1 concurent: 70M/s

## Your Environment

* Version used (`minio --version`):
  minio version RELEASE.2022-02-07T08-17-33Z

* Server setup and configuration:
  "3 nodes with 10 disks per node"
  MINIO_VOLUMES="http://minio{1...3}:9000/mnt/disk{1...10}/minio
  MINIO_STORAGE_CLASS_STANDARD=EC:2

  "3 nodes with 2 disks per node"
   MINIO_VOLUMES="http://minio{1...3}:9000/mnt/disk{1...2}/minio
  MINIO_STORAGE_CLASS_STANDARD=EC:2

* Operating System and version (`uname -a`):
  Linux 3.10.0-862.el7.x86_64 #1 SMP Fri Apr 20 16:44:24 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by klauspost** [comment]
You can see this if your drives have different speeds, but otherwise it should not happen. I suggest you write us at `hello@min.io` and we can go into analyzing it.

(should've been in Q&A - doesn't seem like you can change that for whatever reason)

**Reply by ravindk89** [reply]
Also using a MinIO version more than 1 year old, on Kernel 3.10 which is very EOL.

If you are going to benchmark, please use latest stable MinIO and a non-EOL Linux Kernel + OS.
